### DISCUSSION TITLE
parallel computing

### DISCUSSION THREAD

**Discussion by CPLA81192** [Anura3D/Anura3D_OpenSource#147] [Q&A] [answered]
Can the software compute in parallel? I only have a single core CPU working in the calculation process, resulting in very low computing efficiency!

**Comment by murphy-john** [comment, marked as answer]
Unfortunately, no Anura is only running on one thread at the moment. We hope to implement some parallelization in the future.

**Reply by HaoranzzzZZZ** [reply]
Hi, does Anura3D support parallel computing now?

**Reply by 19Dario88** [reply]
I'm interested in the same topic. I observed that in the .CPS file we can change the number of threads, but I don't know if it means that code support parallelization...

**Reply by zrui24910-crypto** [reply]
> I'm interested in the same topic. I observed that in the .CPS file we can change the number of threads, but I don't know if it means that code support parallelization...

Hi, I looked through the .CPS file carefully but couldn't find an option to modify the number of threads. Could you share which version you're using? Also, did changing this parameter improve the calculation efficiency?
